pkg/proxy/ipvs/proxier.go

```diff
@@ -1352,8 +1387,16 @@ func (proxier *Proxier) syncProxyRules() {
 				}
 				switch protocol {
 				case "tcp":
+					if valid := proxier.nodePortSetTCP.validateEntry(entry); !valid {
```

Comment:
This can be done tighter/cleaner with:

var string name
switch protocol {
   case "tcp":
      name = ...
...
}
if valid := .... {
...
}
```

Reply:
Done.